R/utils.R

```diff
@@ -49,3 +49,13 @@ assert_text <- function(text) {
   }
   text
 }
+
+#' Find the index of the next non-comment in a parse table
+#' @param pd A parse table.
+#' @param pos The position of the token to start the search from.
+#' @importFrom rlang seq2
+next_non_comment <- function(pd, pos) {
+  if (pos >= nrow(pd)) return(integer(0))
```

Comment:
This statement seems redundant.

Reply:
Yes, in the current use context it seems, so I can remove it. Just wanted to add safety.

Reply:
Yes, in the current use context it seems, so I can remove it. Just wanted to add safety.